src/components/Articles/ReadUpdateArticleComponent.jsx

```diff
@@ -11,6 +11,10 @@ import Bookmark from '../../containers/Articles/BookmarkContainer';
 import CommentsContainer from '../../containers/Comments/CommentsContainer';
 import LikeDislike from '../../containers/Articles/LikeDislikeContainer';
 import SocialShare from './ShareComponent';
+import '../../assets/styles/Highlight.scss';
+// eslint-disable-next-line
```

Comment:
Instead of disabling the eslint check (which I assume is about sorting imports), just sort the imports correctly